Index: b/extensions/libxt_multiport.man
===================================================================
--- a/extensions/libxt_multiport.man	2009-01-15 16:14:54.407588160 -0500
+++ b/extensions/libxt_multiport.man	2009-01-15 16:15:45.498588683 -0500
@@ -5,7 +5,7 @@
 or
 .BR "\-p udp" .
 .TP
-[\fB!\fP] \fB\-\-source-ports\fP,\fB\-\-sport\fP \fIport\fP[\fB,\fP\fIport\fP|\fB,\fP\fIport\fP\fB:\fP\fIport\fP]...
+[\fB!\fP] \fB\-\-source-ports\fP,\fB\-\-sports\fP \fIport\fP[\fB,\fP\fIport\fP|\fB,\fP\fIport\fP\fB:\fP\fIport\fP]...
 Match if the source port is one of the given ports.  The flag
 .B \-\-sports
 is a convenient alias for this option. Multiple ports or port ranges are
@@ -13,7 +13,7 @@
 \fB53,1024:65535\fP would therefore match ports 53 and all from 1024 through
 65535.
 .TP
-[\fB!\fP] \fB\-\-destination-ports\fP,\fB\-\-dport\fP \fIport\fP[\fB,\fP\fIport\fP|\fB,\fP\fIport\fP\fB:\fP\fIport\fP]...
+[\fB!\fP] \fB\-\-destination-ports\fP,\fB\-\-dports\fP \fIport\fP[\fB,\fP\fIport\fP|\fB,\fP\fIport\fP\fB:\fP\fIport\fP]...
 Match if the destination port is one of the given ports.  The flag
 .B \-\-dports
 is a convenient alias for this option.
